components/trezor_bridge/resources/trezor/iframe.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
```

Comment:
If this can be removed (see comment about UI, below), then we should rename this file from `iframe.tsx` to `iframe.ts`

Reply:
done